extensions/wikia/Flags/models/Flag.class.php

```diff
@@ -326,4 +324,17 @@ private function verifyParamsForRemove( $flags ) {
 
 		return true;
 	}
+
+	/**
+	 * Queue task for logging flag change
+	 * @param array $flags list of flags changed, each item of that list is an array with flag fields as items
+	 * @param int $pageId ID of article where flags were changed
+	 * @param string $action Type of action performed on flag represented by constants in Flags\Models\Flag class
+	 */
+	private function logFlagChange( $flags, $pageId, $action ) {
```

Comment:
Can you rename `$action` to `$actionType` to be consistent with the `FlagsLogTask::logFlagChange`?